src/dice/hash/blake/internal/blake3/CMakeLists.txt

```diff
@@ -0,0 +1,254 @@
+cmake_minimum_required(VERSION 3.9 FATAL_ERROR)
+
+# respect C_EXTENSIONS OFF without explicitly setting C_STANDARD
+if (POLICY CMP0128)
+  cmake_policy(SET CMP0128 NEW)
+endif()
+
+project(libblake3
+        VERSION 1.5.4
+        DESCRIPTION "BLAKE3 C implementation"
+        LANGUAGES C ASM
+)
+
+include(FeatureSummary)
+include(GNUInstallDirs)
+
+set(blake3_rev 95e42b84fc4709974c7b23c7ae885989ab36c31e)
+set(blake3_files
+        blake3.c
+        blake3.h
+        blake3-config.cmake.in
+        blake3_avx2.c
+        blake3_avx2_x86-64_unix.S
+        blake3_avx2_x86-64_windows_gnu.S
+        blake3_avx2_x86-64_windows_msvc.asm
+        blake3_avx512.c
+        blake3_avx512_x86-64_unix.S
+        blake3_avx512_x86-64_windows_gnu.S
+        blake3_avx512_x86-64_windows_msvc.asm
+        blake3_dispatch.c
+        blake3_impl.h
+        blake3_neon.c
+        blake3_portable.c
+        blake3_sse2.c
+        blake3_sse2_x86-64_unix.S
+        blake3_sse2_x86-64_windows_gnu.S
+        blake3_sse2_x86-64_windows_msvc.asm
+        blake3_sse41.c
+        blake3_sse41_x86-64_unix.S
+        blake3_sse41_x86-64_windows_gnu.S
+        blake3_sse41_x86-64_windows_msvc.asm
+        libblake3.pc.in
+)
+set(blake3_license_files
+        LICENSE_A2
+        LICENSE_A2LLVM
+        LICENSE_CC0
+)
+
+foreach(file ${blake3_files})
+  file(DOWNLOAD "https://raw.githubusercontent.com/BLAKE3-team/BLAKE3/${blake3_rev}/c/${file}"
+          "${CMAKE_CURRENT_BINARY_DIR}/src/blake3/${file}"
+          TLS_VERIFY ON)
+endforeach()
+foreach(file ${blake3_license_files})
+  file(DOWNLOAD "https://raw.githubusercontent.com/BLAKE3-team/BLAKE3/${blake3_rev}/${file}"
+        "${CMAKE_CURRENT_BINARY_DIR}/src/blake3/${file}"
+        TLS_VERIFY ON)
+endforeach()
+
+# architecture lists for which to enable assembly / SIMD sources
```

Comment:
The rest of this stuff below is not necessary anymore, right?

Reply:
Blake3 does not include a usable CMakeLists.txt itself (the one in their repo only runs test cases).
So our CMakeLists.txt ist still needed.

Reply:
Ah forgot about that, can you add a comment so that we don't wonder about it later?